src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -282,6 +283,23 @@ public virtual ClaimsIdentity CreateClaimsIdentity(SecurityToken securityToken,
         }
 
         /// <summary>
+        /// Users can override the default <see cref="CryptoProviderFactory"/> with this property. This factorty will be used for creating signature providers.
+        /// </summary>
+        public CryptoProviderFactory CryptoProviderFactory
+        {
+            get
+            {
+                return _cryptoProviderFactory;
+            }
+            set
+            {
+                if (value == null)
+                    throw LogHelper.LogException<ArgumentNullException>(LogMessages.IDX10001, nameof(CryptoProviderFactory));
+                _cryptoProviderFactory = value;
```

Comment:
our standard style is to have a line after a clause, in this case it is an 'if'.